BlockchainSdk/BlockchainService/XRP/XrpTarget.swift

```diff
@@ -9,13 +9,42 @@
 import Foundation
 import Moya
 
-enum XrpUrl: String {
-    case xrpLedgerFoundation = "https://xrplcluster.com/"
-    case ripple = "https://s1.ripple.com:51234"
-    case rippleReserve = "https://s2.ripple.com:51234/"
+enum XrpUrl {
+    case xrpLedgerFoundation
+    case nowNodes(apiKey: String)
+    case getBlock(apiKey: String)
     
     var url: URL {
-        return URL(string: rawValue)!
+        switch self {
+        case .xrpLedgerFoundation:
+            return URL(string: "https://xrplcluster.com/")!
+        case .nowNodes(let apiKey):
+            return URL(string: "https://xrp.nownodes.io")!
+        case .getBlock(let apiKey):
```

Comment:
Чтобы не было предупреждения

```suggestion
        case .nowNodes:
            return URL(string: "https://xrp.nownodes.io")!
        case .getBlock:
```